### DISCUSSION TITLE
[Suggestion] Custom Skins

### DISCUSSION THREAD

**Discussion by zalexville** [tsunamayo/Starship-EVO#4018] [Suggestions]
**Idea**
One thing that could make this game superior to others in the field would be the ability to make custom character skins. Maybe we do not need to have custom heads/bodies (though if possible that would be cool), but the ability to design a custom uniform or even the ability to change primary/secondary colors would be amazing.

**Comment by Kaiser-Indrasil** [comment]
Agreed, being able to change the colour of our uniforms would be very cool as a vanilla feature. I imagine the more advanced features could be delegated to mods.

**Comment by RockefellerDoctor** [comment]
Considering the game is still in it's general proof of concept phase, I'd rather he focus on nailing down mechanics before messing around with cosmetics.
